docs/releases/upgrading/upgrade-v2.0.md

```diff
@@ -38,3 +38,44 @@ pipenv run invenio rdm-records rebuild-index
 ~~~
 
 Alternatively, just download and copy the [upgrade script](./scripts/upgrade-rdm-1.0-to-2.0.sh) into the instance directory, and execute it.
+
+## Troubleshooting
+
+This Troubleshooting guide is intended to solve known issues that might appear after upgrading from v1.0 to v2.0.
+
+### Prerequisites
+
+Have executed an upgrade from v1.0 to v2.0 following the instructions in the previous section.
+
+### Resolving known errors
+
+**Error deleting a draft**
+
+*Problem:* When deleting a draft an SQLAlchemy error will happen in the server and it won't work.
+
+*Cause:* This is caused by an Invenio module (invenio-pidrelations) that it's not used anymore and is a left over from previous version.
+
+*Solution:* Remove the unused module. This can be achieved by the following shell command:
+
+~~~bash
+pipenv run pip uninstall invenio-pidrelations
+~~~
+
+**Record versions error**
+
+*Problem:* When creating a new version on a migrated record, the second version appears as "Version v1".
+
+*Cause:* This is caused by an incomplete migration of the records.
```

Comment:
If we do a new release of v2 including the [small fix](https://github.com/inveniosoftware/invenio-app-rdm/pull/803) in the upgrade script we would prevent this issue and we could change the text here to: 
```suggestion
*Cause:* This is caused by an incomplete migration of the records (fixed in v.2.0.2 of invenio-app-rdm).
```